halo2_proofs/tests/frontend_backend_split.rs

```diff
@@ -577,8 +580,11 @@ fn test_mycircuit_full_split() {
     let start = Instant::now();
     let mut witness_calc = WitnessCalculator::new(k, &circuit, &config, &cs, instances_slice);
     let mut transcript = Blake2bWrite::<_, G1Affine, Challenge255<_>>::init(vec![]);
+    // TODO: is ProverV2Single::new part of the public API?
```

Comment:
Yes, `ProverV2Single::new` is part of the public API.  Please consider my previous comment about having `ProverV2` hold a reference to the engine passed via the constructor so that we can call the `commit_phase` and `create_proof` without passing the engine again.